streams.yml

```
@@ -59,6 +59,13 @@ rhel8:
   upstream_image: registry.ci.openshift.org/ocp/builder:rhel-8-openshift-{MAJOR}.{MINOR}.art
   mirror: true
 
+rhel-7-golang:
+  image: openshift/golang-builder:v1.18
```

```suggestion
  image: openshift/golang-builder:v1.18.10-202311291919.el7.g87f8b77
```

Completed build: [openshift-golang-builder-container-v1.18.10-202311291919.el7.g87f8b77](https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=2800175)